http/get_simple/rs/client/src/main.rs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use arrow_ipc::reader::StreamReader;
+use std::{
+    io::{BufRead, BufReader, Write},
+    net::{IpAddr, Ipv4Addr, SocketAddr, TcpStream},
```

Comment:
Can we make the client print elapsed time?

Feel free to ignore my suggestions here and do it differently :) Thanks
```suggestion
    net::{IpAddr, Ipv4Addr, SocketAddr, TcpStream},
    time::{Instant},
```

Reply:
I added another span around the full example. With `.with_span_events(FmtSpan::CLOSE)` the `fmt` subscriber outputs the span duration:
```
2024-03-08T18:20:22.956542Z  INFO get_simple:Reading Arrow IPC stream{addr=127.0.0.1:8000}: client: Connected
2024-03-08T18:20:23.617011Z  INFO get_simple:Reading Arrow IPC stream{addr=127.0.0.1:8000}: client: batches=24415 rows=100000000
2024-03-08T18:20:23.748391Z  INFO get_simple:Reading Arrow IPC stream{addr=127.0.0.1:8000}: client: close time.busy=792ms time.idle=666ns
2024-03-08T18:20:23.748413Z  INFO get_simple: client: close time.busy=792ms time.idle=708ns
```

Note the `close time.busy=792ms time.idle=708ns` part.